packages/core/src/components/Table/stories/storiesUtils.tsx

```diff
@@ -187,23 +187,45 @@ export const getGroupedColumns = (): HvTableColumnConfig<
   AssetEvent,
   string
 >[] => [
-  { Header: "Title", accessor: "name", style: { minWidth: 120 } },
-  { Header: "Time", accessor: "createdDate", style: { minWidth: 100 } },
-  { Header: "Event Type", accessor: "eventType", style: { minWidth: 100 } },
   {
+    id: "gc-title",
```

Comment:
I added the `id`s because the grouped column doesn't have an accessor and I was getting pa11y errors since the `id` used was "Event Info" (the `Header`) and had blank spaces.
I'll remove the `id`s and only use it for the event info.